go.mod

```diff
@@ -17,13 +17,17 @@ require (
 	github.com/spf13/cast v1.5.0
 	github.com/spf13/cobra v1.6.1
 	github.com/spf13/pflag v1.0.5
+	github.com/strangelove-ventures/packet-forward-middleware/v7 v7.0.0-20230523193151-73dea436e53f
```

Comment:
@DongLieu can you bump this to the correct version? Why is it on a specific commit hash?

Reply:
Currently, i did not find the official one for sdk v0.47.2. If you want I can make a fork version.